packages/react-data-grid/src/connectedComponents/SelectionMask.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Motion, spring } from 'react-motion';
+import { connect } from '../stateManagement/state/RxState';
+import initialState from '../stateManagement/state/initialState';
+
+/* MapStateToProps */
+
+const Mask = ({ selectedPosition }) => {
+  const { idx, rowIdx } = selectedPosition;
+  const [width, height] = [150, 35];
```

Comment:
assuming fixed cell dimensions to calculate position. Need to figure out the optimal way to obtain this info